### PR TITLE
Update the installation part of the documentation.

### PR DESCRIPTION
**Pull request by cwwalter**
Updating the installation part of the sphinx code.

### CODE REVIEW

**Comment by cwwalter** [comment]
Jim,  you don't need to look at the documentation changes I made here.

But, there are changes to the CI file (and conda standalone requirements) I made so that things worked with a recent version of the stack/python.

Can you take a look at and review that? Thanks.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -49,9 +49,6 @@ jobs:
 
             - name: Install pip deps
               run: |
-                # We need to get batoid onto conda, but for now, this is a separate step.
-                pip install batoid
-                pip install batoid_rubin
```
Comment (jchiang87) [inline .github/workflows/ci.yml:54]: Just to be clear: we don't need these because they are available from the conda distribution now?

Comment (cwwalter) [reply .github/workflows/ci.yml:54]: That's right. I added them to the standalone-requirements file.

Actually we don't need rubin-sim either.  But, I kept it this way so as to not download the parts we need.

**Review by jchiang87 — APPROVED** [review]
LGTM.

**Pull request merged**